joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/NodeBuilder.scala

```diff
@@ -65,13 +65,13 @@ class NodeBuilder(diffGraph: DiffGraphBuilder) {
     addNodeToDiff(typeRefNode)
   }
 
-  def memberNode(name: String, dynamicTypeHintFullName: String): nodes.NewMember = {
+  def memberNode(name: String, dynamicTypeHintFullName: Option[String] = None): nodes.NewMember = {
```

Comment:
I would suggest providing two `memberNode` methods.
1. `memberNode(String, String): nodes.NewMember`
2. `memberNode(String): nodes.NewMember`

That way we dont have to wrap the second argument in `Option`/`Some`.